configs/configs_lidro.yaml

```
@@ -47,31 +53,26 @@ virtual_point:
     # The number of nearest neighbors to find with KNeighbors
     k: 20
 
-SKELETON:
-  FILE_PATH:
-    MASK_INPUT_PATH:
-    SKELETON_LINES_OUTPUT_PATH:  # only branches' skeleton (if empty, no file is created)
-    GAP_LINES_OUTPUT_PATH:  # only new lines to close gaps (if empty, no file is created)
-    GLOBAL_LINES_OUTPUT_PATH:  # lines from both the skeletons and the gap lines
-
-  MAX_GAP_WIDTH: 200 # distance max in meter of any gap between 2 branches we will try to close with a line
-  MAX_BRIDGES: 1 # number max of bridges that can be created between 2 branches
-  GAP_WIDTH_CHECK_DB: 20 # for a gap at least this wide, we check with DB_UNI if there is a bridge. Smaller than that, we assume a line automatically exists
-  RATIO_GAP: 0.25 # when checking if a candidate line to close a gap intersects a bridge in BD UNI, that's the ratio ]0.1] of the line to consider.
+skeleton:
+  max_gap_width: 200 # distance max in meter of any gap between 2 branches we will try to close with a line
+  max_bridges: 1 # number max of bridges that can be created between 2 branches
+  gap_width_check_db: 20 # for a gap at least this wide, we check with db_uni if there is a bridge. Smaller than that, we assume a line automatically exists
+  ratio_gap: 0.25 # when checking if a candidate line to close a gap intersects a bridge in BD UNI, that's the ratio ]0.1] of the line to consider.
                   # we only check part of the line, because when the line is really long it can intersect with bridges not corresponding to that line,
                   # so we make the line smaller
 
-  DB_UNI:
-    DB_NAME: bduni_france_consultation
-    DB_HOST: bduni_consult.ign.fr
-    DB_USER:
-    DB_PASSWORD:
-    DB_PORT: 5432
+  db_uni:
+    db_using_db: True # if false, the DB won't be queried
```

Pinaillage : pour rajotuer du contexte, ça porurait devenir : 
```suggestion
    db_using_db: True # if false, the DB won't be queried (expected to be used only in case you don't have access to the database) 
```

---

Ok, je vais justele mettre de mon côté pour mettre la même chose dans le readme